pages/result.tsx

```diff
@@ -0,0 +1,19 @@
+import { Grid, Text } from '@chakra-ui/react';
+
+import ResulData from '@packages/components/ResultData';
+
+import type { NextPage } from 'next';
+
+const ResultPage: NextPage = () => {
+  return (
+    <Grid h="100vh" w="100vw" justifyContent="center">
```

Comment:
You're not using grid features here, maybe you should replace it with something like Box

Reply:
When I apply Box, it loses all properties.

Reply:
Yeah, it will because you need to configure the div to hold the elements you're rendering. My point here is not that. Is that you're using the GRID structure withoud really using it, we should keep our code as simple as possible and use complex structures only when needed.

In the future, I would look at the code and stay some time thinking and considering, "Why is this Grid here ?" and would take some debugging just to understand that it servers no pourpuse. Does this make sense?